.github/workflows/dockerhub-publish.yml

```diff
@@ -0,0 +1,54 @@
+name: dockerhub-publish
+
+on:
+ push:
+   tags: 
+    - 'v[0-9]+.[0-9]+.[a-zA-Z0-9]+'
```

Comment:
If this is a regex, shouldn't the periods be escaped? ie `v[0-9]+\.[0-9]+\.[a-zA-Z0-9]+`. Otherwise they would match any.

Also does github actions regex allow `[A-z]` in place of `[a-zA-Z]`?